paddle/operators/name_convention.md

```diff
@@ -0,0 +1,11 @@
+## Operator Name Convention
+
+To make the operator document itself more clear. we recommend operator names observe the listing conventions.
+
+### Input/Output names
+
+Variable name is uppercase. e.g. `X`, `Y`
+
+Tensor name is lowercase. e.g. `tensor`
```

Comment:
Operator's inputs/outputs are `Variable` not `Tensor`.
So what's the meaning of `Tensor name`?